.github/workflows/ci.yml

```diff
@@ -238,25 +304,47 @@ jobs:
             - name: Run docker
               run: |
                   git clone https://github.com/spryker/docker-sdk.git ./docker
-                  docker/sdk boot deploy.ci.functional.mariadb.yml -v
-                  docker/sdk up -t -v
+                  docker/sdk boot deploy.ci.functional.mariadb.yml
+                  docker/sdk up -t
                   docker/sdk testing codecept run -c codeception.functional.yml
 
+    php-81-mariadb-glue-alpine-dynamic-store-off:
+        name: "PHP 8.1 / MariaDB / Glue / Alpine / Dynamic Store OFF"
+        runs-on: ubuntu-20.04
+        env:
+            PROGRESS_TYPE: plain
+            SPRYKER_PLATFORM_IMAGE: spryker/php:8.1
+            TRAVIS: 1
+
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Install docker-compose
+              run: |
+                  sudo curl -L "https://github.com/docker/compose/releases/download/2.12.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+                  sudo chmod +x /usr/local/bin/docker-compose
+
+            - name: Run docker
+              run: |
+                  git clone https://github.com/spryker/docker-sdk.git ./docker
+                  docker/sdk boot deploy.ci.api.mariadb.dynamic-store-off.yml
+                  docker/sdk up -t
+                  docker/sdk testing codecept fixtures
+                  docker/sdk testing console queue:worker:start --stop-when-empty
+                  docker/sdk testing codecept run -c codeception.api.yml
```

Comment:
* Before the test execution we can run the "docker/sdk cli composer dump-autoload -o -a --apcu" will speed up the execution. (Should be used for all jobs).